components/_patterns/02-molecules/about-x/_about-x.scss

```diff
@@ -0,0 +1,20 @@
+$about-x--break: $medium; // This is a random breakpoint. Design should verify it.
+
+.about-x__content {
+  display: flex;
+  flex-flow: column nowrap;
+
+  @include breakpoint($about-x--break) {
+    flex-direction: row;
+  }
+}
+
+.about-x__image {
+  margin-bottom: 1rem;
+
+  @include breakpoint($about-x--break) {
+    margin-bottom: 0;
+    margin-right: 2rem;
+    flex: 0 0 164px;
```

Comment:
This number feels a little odd. We haven't talked about grids yet but looking at the grid it looks like the designers wanted the image to take up two columns: 

<img width="827" alt="Screen Shot 2019-09-10 at 9 51 24 AM" src="https://user-images.githubusercontent.com/1720093/64619729-904a3d00-d3b0-11e9-887e-5fb2d1522b4b.png">

Since the item takes up 9 overall, I think we want something that observes that 2/9 ratio (this isn't perfect, obviously, since it doesn't account for gutters). 2/9s is also super funky, so we could do this as 1/5 to keep it simple and respect that ratio in other places it's used (looks like catalog list items is the only other one with this particular ratio, every other organism seems to be 1/9 or 3/9 when I eyeball it.)

Reply:
We talked about this via zoom, so I'm resolving it here.